portals/publisher/source/src/app/components/Apis/Details/ProductResources/ProductResourcesEdit.jsx

```diff
@@ -127,6 +127,7 @@ function ProductResourcesEdit() {
                         <Grid item>
                             <div>
                                 <CustomSplitButton
+                                    api={api}
```

Comment:
use a null check for advertiseInfo

Reply:
Done with https://github.com/wso2/apim-apps/pull/130/commits/bab2439306d03e8f1775c16d276683ba43e558dd